internal/ctrlutils/ingress-status.go

```diff
@@ -212,11 +214,15 @@ func UpdateIngress(
 				break
 			}
 			if errors.IsNotFound(err) {
+				log.Debugf("failed to update Ingress V1 status because the object (%s/%s) is gone, status update stopped.", namespace, name)
 				return nil
 			}
-
-			log.Errorf("failed to update Ingress V1 status. %v. retrying...", err)
-			time.Sleep(time.Second)
+			if errors.IsConflict(err) {
+				log.Debugf("failed to update Ingress V1 status because the object (%s/%s) changed. retrying...", namespace, name)
```

Comment:
```suggestion
				log.Debugf("failed to update Ingress V1 (%s/%s) status, conflict %v", namespace, name, err)
```

Reply:
:+1: changed in 2812019f36e6aed24e3021ec977f0475ab68e90d